test/es-module/test-esm-cjs-main.js

```diff
@@ -24,4 +24,5 @@ child.stdout.on('data', (data) => {
 child.on('close', common.mustCall((code, stdout) => {
   assert.strictEqual(validatedExecution, true);
   assert.strictEqual(code, 0);
+  assert.strictEqual(stdout, null);
```

Comment:
We should rename `stdout` -- parameters to the `close` event are `code` and `signal`:
https://nodejs.org/dist/latest-v11.x/docs/api/child_process.html#child_process_event_close